xds/src/test/java/io/grpc/xds/XdsClientWrapperForServerSdsTest.java

```diff
@@ -139,6 +140,26 @@
   private XdsClientWrapperForServerSds xdsClientWrapperForServerSds;
   private final DownstreamTlsContext[] tlsContexts = new DownstreamTlsContext[3];
 
+  /** Creates XdsClientWrapperForServerSds: also used by other classes. */
+  public static XdsClientWrapperForServerSds createXdsClientWrapperForServerSds(
```

Comment:
For now I am leaving it here as the "container" for `XdsClientWrapperForServerSds` related util methods instead of creating a new util class in this package.